File: week 3 - 3. navctrl w:SQLite/NavCtrl/NavCtrl/Product.m
//
//  Product.m
//  NavCtrl
//
//  Created by Julio Reyes on 3/2/15.
//  Copyright (c) 2015 Julio Reyes. All rights reserved.
//

#import "Product.h"

@implementation Product

- (id)initWithID:(NSInteger)productid
        ProductName:(NSString *)productname
        ProductReference:(NSString *)productref
        CompanyID:(NSInteger)companyid
{
    self = [super init];
    if (self) {
        self.ProductID = productid;
        self.ProductName = productname;
        self.ProductReference = productref;
        self.CompanyID = companyid;
    }
    return self;
}
@end

File: Swift/navCtrl/navCtrl/Product.swift
//
//  Product.swift
//  navCtrl
//
//  Created by Aditya Narayan on 2/19/15.
//  Copyright (c) 2015 JulioReyesCorps. All rights reserved.
//

import Foundation

class Product {
    var id:Int
    var name:String
    var companyId:Int?
    
    init(id:Int, name:String){
        self.id = id;
        self.name = name
    }
}
